project/src/main/java/com/google/sps/data/ServletLibrary.java

```diff
@@ -344,6 +344,9 @@ public static void addProductToBusiness(DatastoreService datastore, String produ
         System.err.println("Entity properties are of an incorrect type.");
         return null;
       }
+      @SuppressWarnings("unchecked") // Documentation says to suppress warning this way
+        List<String> gcsUrls = (ArrayList<String>) entity.getProperty("gcsUrls"); 
+      if (gcsUrls == null) gcsUrls = new ArrayList<String>();
       @SuppressWarnings("unchecked") // Documentation says to suppress warning this way
         List<String> imageUrls = (ArrayList<String>) entity.getProperty("imageUrls"); 
```

Comment:
Should these lines be indented under the annotations?  The official Google style guide basically says you can do whatever you want with local variable annotations so it's up to you.

Reply:
The documentation for datastore also had the indentation under the suppressed warning, so I just kept it that way for clarity to know that we are suppressing the next line's casting.